src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import ArticleCard from './ArticleCard';
+import './ArticleList.css';
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get('https://jesjin-nc-news.onrender.com/api/articles')
```

Comment:
You can use your fetchArticles function here instead of repeating the same logic!